encrypted-media-respec.html

```diff
@@ -674,7 +674,7 @@ <h3><dfn>Navigator</dfn> Extension: <code>requestMediaKeySystemAccess()</code></
               Otherwise, it is resolved with a new <a>MediaKeySystemAccess</a> object.
             </p>
             <div class="note">
-              <p>This method is only exposed to <a href="https://www.w3.org/TR/secure-contexts/#secure-context">secure contexts</a> [[SECURE-CONTEXTS]] as indicated by the <code>[SecureContext]</code> IDL attribute.</p>
+              <p>This method is only exposed to <a href="https://www.w3.org/TR/secure-contexts/">secure contexts</a> [[SECURE-CONTEXTS]] as indicated by the <code>[SecureContext]</code> IDL attribute.</p>
```

Comment:
Actual definition of the secure context" concept is in HTML and can be reached through `[=secure contexts=]` in Respec (and the reference to the Secure Contexts spec could actually be dropped):

```suggestion
              <p>This method is only exposed to [=secure contexts=] as indicated by the <code>[SecureContext]</code> IDL attribute.</p>
```